model/fn-execution/src/main/proto/org/apache/beam/model/fn_execution/v1/beam_fn_api.proto

```diff
@@ -761,6 +761,8 @@ message Elements {
   // extensible and backward compatible
   message ElementMetadata {
     optional DrainMode.Enum drain = 1;
+    optional string traceparent = 2;
```

Comment:
Can some comments be added on what they are and/or link to doc? I'm guessing parent is some id? but what is trace state? Should it be an enum? Should either of these be bytes?